site/pages/create-consequence-stops.page.tsx

```diff
@@ -135,6 +148,40 @@ const CreateConsequenceStops = ({
         }
     };
 
+    const getMarkers = () => {
+        const stopsInTable =
+            pageState.inputs.stops && pageState.inputs.stops.length > 0
+                ? pageState.inputs.stops.map((stop) => (
+                      <Marker
```

Comment:
It might be nicer if the Map component just takes two lists of stops (selected and searched or something similar) and then the map component can be responsible for creating the Marker components

Reply:
done